### DISCUSSION TITLE
Allow steps component to use elements instead of string data-content

### DISCUSSION THREAD

**Discussion by mateoKutnjak** [saadeghi/daisyui#2109] [Ideas / feature requests]
For example, I would like to put <Icon/> element inside the step instead of character.

**Comment by saadeghi** [comment]
You can do something like this: https://play.tailwindcss.com/Lr3d1R4kHZ

I'm working on a new `timeline` class name that would allow this, with a similar visual style.  
The current `steps` component was designed for a to show that minimal 1 > 2 > 3 > ... steps with the minimum amount of HTML.
